server/webhook_http.go

```diff
@@ -104,7 +104,7 @@ func (p *Plugin) httpWebhook(w http.ResponseWriter, r *http.Request, instanceID
 		return respondErr(w, appErr.StatusCode, appErr)
 	}
 
-	wh, err := ParseWebhook(bb)
+	wh, err := ParseWebhook(bb, p, instanceID)
```

Comment:
We generally try to avoid passing `Plugin` around smaller operations because you can do pretty much anything with it.  Since we're passing `Plugin` and `instanceID` through a bunch of functions just to get the epic name, I propose we abstract this into a `WebhookHelper` struct.

```go
type WebhookHelper struct {
     p *Plugin
     instanceID string
}

func (h *WebhookHelper) GetEpicName(issueID string) (string, error)
```

Then this code becomes

```suggestion
	helper := WebhookHelper{p, instanceID}
	wh, err := ParseWebhook(bb, helper)
```